.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
+          - '1.6'
```

Comment:
Why 1.6 and not v1.5, since that's the oldest version you claim to be compatible with?
https://github.com/JuliaPlots/Plots.jl/blob/b1c56126fb777f9f3b09251f745b51565bb627cd/Project.toml#L58